src/addrdb.h

```diff
@@ -60,10 +60,15 @@ void DumpAnchors(const fs::path& anchors_db_path, const std::vector<CAddress>& a
 
 /**
  * Read the anchor IP address database (anchors.dat)
- *
- * Deleting anchors.dat is intentional as it avoids renewed peering to anchors after
- * an unclean shutdown and thus potential exploitation of the anchor peer policy.
```

Comment:
This property is being lost here in the case where the anchor causes a crash between reading it and making a connection successfully.

Reply:
Yes, make sense.